### PR TITLE
Translate Chinese ( Traditional )

### PR DESCRIPTION
**Pull request by crystalhoo**
- [ 🗸] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [ 🗸 ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
Translation to Chinese (Traditional).

### CODE REVIEW

File: _data/locales/tra-ch.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+tra-ch:
```
Comment (mlinksva) [inline _data/locales/tra-ch.yml]: I think this should be `zh-hant` (or `zh-tw` if you don't mind more common and jurisdiction-denoting language code--the existing simplified Chinese translation uses `zh-cn`). I've never seen `tra-ch` or similar before.

Comment (crystalhoo) [reply _data/locales/tra-ch.yml]: Alright. Thank you for your guiding.

**Comment by jclub020434** [comment]
😇

**Comment by MikeMcQuaid** [comment]
Hey, thanks for the PR! Need to fix the test failures here: 
https://travis-ci.org/github/opensource.guide/builds/410079074

**Comment by mahdibakhtdar** [comment]
Hey,Katrina.
my gmail is  Bakhtdar7@gmail.com;
Are you understand?

**Comment by MikeMcQuaid** [comment]
@crystalhoo Ping on the test failures.

**Comment by mlinksva** [comment]
A traditional Chinese translation was subsequently completed in #962. It looks like a few of the translations started here are slightly different. I'd suggest a new PR with edits to the live translation if there are corrections or improvements to be made. Thanks!

**Pull request closed without merging**